packages/auto_mappr/example/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   equatable: ^2.0.7
 
 dev_dependencies:
-  auto_mappr: ^2.6.0
-  build_runner: ^2.0.0
+  auto_mappr: ^2.7.0
+  build_runner: ^2.4.14
```

Comment:
I would not increase this version of build_runner so we dont enforce ^2.4.14 on apps that use this package.